Source/UI/NewUI/CommonUI/View/Component/InputView.swift

```diff
@@ -86,9 +89,16 @@ class InputView: UIView {
         textFieldContainer.layer.cornerRadius = style.textfield.cornerRadius
         textFieldContainer.layer.borderWidth = style.textfield.borderWidth
         textFieldContainer.backgroundColor = style.textfield.backgroundColor
+    }
 
+    private func updateTextFieldContainer(image: UIImage?, withAnimation: Bool) {
```

Comment:
I'll make this more fun and add that UIViewController().present method uses `animated` 🙃. 

I would throw in more points that:
- withAnimation sounds like we will provide a completion handler for the animation action (similar to `UIView.animate(withDuration`
- `isAnimated` would sound like a stored property that describes the existing state rather than an instruction for how to perform something
- `shouldAnimate` would be syntactically more on point but a less concise option



Either way, whatever you decide (I'm not convinced there's an obvious right / wrong here), please make sure to keep consistency with the publicly exposed declaration at line 67
```func update(style: CellTextFieldStyle? = nil, textFieldValue: String? = nil, image: UIImage? = nil, withAnimation: Bool = false){```